src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java

```diff
@@ -167,27 +168,28 @@ private BlockPos fillProxyList(@NotNull BlockPos target, double distanceToPath)
             proxyPoint = getProxy(target, worker.getPosition(), distanceToPath);
         }
 
-        if(!proxyList.isEmpty())
+        if (!proxyList.isEmpty())
         {
             proxyList.remove(0);
         }
 
-        Log.getLogger().info("Path contains :" + (proxyList.size()+1) + " pathpoints");
+        Log.getLogger().info("Path contains :" + (proxyList.size() + 1) + " pathpoints");
         Log.getLogger().info("First target :" + proxyPoint);
         return proxyPoint;
     }
 
     /**
      * Returns a proxy point to the goal for the miner especially.
-     * @param target the target.
+     *
+     * @param target         the target.
      * @param distanceToPath the total distance.
      * @return a proxy or, if not applicable null.
      */
     @NotNull
     private BlockPos getMinerProxy(final BlockPos target, final double distanceToPath)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getMinerProxy" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)